app/routes/people/index.js

```diff
@@ -3,6 +3,15 @@ import Route from '@ember/routing/route';
 // eslint-disable-next-line ember/no-mixins
 import DataTableRouteMixin from 'ember-data-table/mixins/route';
 
-export default class PeopleIndexRoute extends Route.extend(DataTableRouteMixin){ 
+export default class PeopleIndexRoute extends Route.extend(DataTableRouteMixin){
   modelName = 'person';
-}
\ No newline at end of file
+
+  mergeQueryOptions() {
+    return {
+      include: [
+        'mandatories.mandate.governing-body',
```

Comment:
Using includes reduces the amount of api calls when rendering the table. If you don't use includes, the table will make a call for each relationship per row so that adds up really fast. (and looks weird since the table updates once the data comes in).

Reply:
You can check the difference by commenting this code and checking the network inspector.